widelands.yaml

```diff
@@ -0,0 +1,54 @@
+name: widelands
+buildsystem: cmake-ninja
+builddir: true
+config-opts:
+  - -DCMAKE_BUILD_TYPE=Release
+  - -DCMAKE_INSTALL_PREFIX=/app/bin
+  - -DWL_INSTALL_BASEDIR=/app/share/widelands
+  - -DWL_INSTALL_DATADIR=/app/share/widelands
+  - -DBOOST_ROOT=/app
+  - -DGLEW_ROOT=/app
+  - -DGLEW_GLEW_LIBRARY=/app/lib/libGLEW.so
+  - -DGLEW_INCLUDE_DIR=/app/include/GL
+  # https://wl.widelands.org/forum/topic/1343/?page=3 & https://wl.widelands.org/forum/post/9857/
+  - -DGLEW_LIBRARY:FILEPATH=/app/lib/libGLEW.a
+  - -DOPENGL_glu_LIBRARY=/app/lib/libGLU.a
+  - -DOPTION_BUILD_WEBSITE_TOOLS=OFF
+sources:
+  # Source0
+  - type: archive
+    url: https://launchpad.net/widelands/build19/build19/+download/widelands-build19-src-gcc7.tar.bz2
+    sha256: ef73b8cf5a7c680bad5caade907f1cc6ee88dd3729470b9214a3c72658585458
+  # Source1 - Numix-Circle - numix-icon-theme-circle-0.1.0-13.git5a11140.el7.src.rpm
+  - type: file
+    path: widelands.svg
+    sha256: 4f860318acb3271b7cf6dcc6ba53fa265977d66894597195bea3b82d35b22335
+  # Patch0
+  - type: patch
+    path: widelands-build19-ppc64le.patch
+    strip-components: 1
+    only-arches:
+      - ppc64le
+    sha256: 6ecebb71541c726c591e15ea693a01beb0173fa2040283153d958df456ce735b
+  # Patch1
+  # https://bugs.launchpad.net/widelands/+bug/1760581
+  - type: patch
+    path: widelands-build19-icu-bidi.patch
+    strip-components: 1
+    sha256: 651f6d7ea6843752032872c4a20014418d52c5fa01dc72d4ede1d4f030206742
+  - type: file
```

Comment:
Again, there is nothing in these three scripts you couldn't embed into the manifest.

Reply:
> Again, there is nothing in these three scripts you couldn't embed into the manifest.

Quoting @TingPing:
> Just have a single script file (not inline)

I'm not going to rewrite the correct manifest over and over again, because you have such a whim.

Reply:
We have already discussed this topic several months ago. The conclusion was to put everything in separate files. Now, however, you are demanding to return to the previous layout. It shows that you don't know what you want. What's more, there are no technical reasons for such changes, which means that I can't treat it seriously. From my perspective, it looks like a temporary whim.

Reply:
We concluded that these scripts are largely overcomplicated and there should be at most one, not multiple files.

Reply:
> We concluded that these scripts are largely overcomplicated and there should be at most one, not multiple files.

I see only one person who complain about that... In this particular case, one script should be enough, but in general, a division into pre-install and post-install actions is required.